### PR TITLE
docs: V2 alpha readme

### PR DESCRIPTION
**Pull request by fforootd**
This adds

- restyled readme
- actions docs from v1
- contrib guide even though its not entirely correct

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,103 +10,102 @@
...
 
-See our [Documentation](https://docs.zitadel.ch/docs/quickstarts/introduction) to get started with ZITADEL quickly. Let us know, if you are missing a language or framework in the [Q&A](https://github.com/zitadel/zitadel/discussions/1717).
+> If you miss something please feel free to engage with us [here](https://github.com/zitadel/zitadel/discussions/1717)
+
+## Why ZITADEL
+
+- [API-first](https://docs.zitadel.com/docs/apis/introduction)
+- Strong audit trail thanks to [event sourcing](https://docs.zitadel.com/docs/concepts/eventstore)
+- [Actions](https://docs.zitadel.ch/docs/concepts/features/actions) to react on events with custom code
+- [Private labeling](https://docs.zitadel.com/docs/guides/customization/branding) for a uniform user experience
```
Comment (mffap) [inline README.md:47]: Should we call this branding?

File: README.md [hunk 2]
```diff
@@ -10,103 +10,102 @@
...
+
+## Features
+
+- Single Sign On (SSO)
+- Passwordless with FIDO2 support
+- Username / Password
+- Multifactor authentication with OTP, U2F
+- [Identity Brokering](https://docs.zitadel.com/docs/guides/authentication/identity-brokering)
+- [Machine-to-machine (JWT profile)](https://docs.zitadel.com/docs/guides/authentication/serviceusers)
+- [Personal Access Tokens (PAT)]()
```
Comment (mffap) [inline README.md:58]: Guide is missing https://github.com/zitadel/zitadel/issues/3678

File: README.md [hunk 3]
```diff
@@ -10,103 +10,102 @@
...
+- Role Based Access Control (RBAC)
+- Delegate role management to third-parties
```
Comment (mffap) [inline README.md:60]:
```suggestion
- [Delegate role management to third-parties](https://docs.zitadel.ch/docs/guides/basics/projects#what-is-a-granted-project)
```

File: README.md [hunk 4]
```diff
@@ -10,103 +10,102 @@
...
-## Help and Documentation
+| Language | Client | API | Machine auth (\*) | Auth check (\*\*) | Thanks to the maintainers |
+|----------|--------|--------------|----------|---------|---------------------------|
+| .NET     | [zitadel-net](https://github.com/zitadel/zitadel-net) | GRPC | ✔️ | ✔️ | [buehler 👑](https://github.com/buehler) |
+| Dart     | [zitadel-dart](https://github.com/zitadel/zitadel-dart) | GRPC | ✔️ | ❌ | [buehler 👑](https://github.com/buehler) |
+| Elixir   | [zitadel_api](https://github.com/jshmrtn/zitadel_api) | GRPC | ✔️ | ✔️ | [jshmrtn 🙏🏻](https://github.com/jshmrtn) |
+| Go       | [zitadel-go](https://github.com/zitadel/zitadel-go) | GRPC | ✔️ | ✔️ | ZITADEL |
+| Rust     | [zitadel-rust](https://crates.io/crates/zitadel) | GRPC | ✔️ | ❌ | [buehler 👑](https://github.com/buehler) |
+| JVM      | ❓ | ❓ | ❓ | | Maybe you? |
+| Python   | ❓ | ❓ | ❓ | | Maybe you? |
```
Comment (mffap) [inline README.md:77]: Link issue? https://github.com/zitadel/zitadel/issues/3675

File: README.md [hunk 5]
```diff
@@ -10,103 +10,102 @@
...
-  * API-first
-  * Actions for custom code execution
-
 ## Showcase
 
+<!-- TODO: Replace Images-->
+
 ### Passwordless Login
 
 Use our login widget to allow easy and sucure access to your applications and enjoy all the benefits of passwordless (FIDO 2 / WebAuthN):
```
Comment (mffap) [inline README.md:97]:
```suggestion
Use our login widget to allow easy and secure access to your applications and enjoy all the benefits of passwordless (FIDO 2 / WebAuthN):
```

File: README.md (hunk 1 above)
Comment (fforootd) [reply README.md:47]:
```suggestion
- [Branding](https://docs.zitadel.com/docs/guides/customization/branding) for a uniform user experience
```

File: README.md (hunk 2 above)
Comment (fforootd) [reply README.md:58]:
```suggestion
- Personal Access Tokens (PAT)
```

File: README.md (hunk 4 above)
Comment (fforootd) [inline README.md:77]:
```suggestion
| JVM      | 🚧 [WIP](https://github.com/zitadel/zitadel/discussions/3650) | ❓ | ❓ | | TBD |
| Python   | 🚧 [WIP](https://github.com/zitadel/zitadel/issues/3675) | ❓ | ❓ | | TBD |
```

Comment (fforootd) [reply README.md]: done

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -31,16 +115,8 @@ Make sure you use semantic release:
...
-Checkout the following chapters to start ZITADEL locally.
-
-### Build for local development
-
-* Check out our [development guide](guides/development.md).
-
-### Quick start
-
-* Check out our [quick start guide](guides/quickstart.md).
+TODO Link to public docs
```
Comment (hifabienne) [inline CONTRIBUTING.md:118]:
```suggestion
You can find an installation guide for all the different environments here:
https://docs.zitadel.com/docs/guides/installation
```

File: README.md [hunk 7]
```diff
@@ -1,112 +1,126 @@
...
+</p>
 
-## What Is ZITADEL
+<p align="center">
+    <a href="https://openid.net/certification/#OPs" alt="OpenID Connect Certified">
+        <img src="./docs/static/logos/oidc-cert.png" /></a>
+</p>
 
-**ZITADEL** is a "Cloud Native Identity and Access Management" solution built for the cloud era. ZITADEL uses a modern software stack consisting of [**Golang**](https://golang.org/), [**Angular**](https://angular.io/) and  [**CockroachDB**](https://www.cockroachlabs.com/) as sole storage and follows an event sourced pattern.
+ZITADEL is the modern open* alternative for Auth0, Firebase Auth, AWS Cognito as well as Keycloak built for the container and serverless era.
```
Comment (hifabienne) [inline README.md:30]:
```suggestion
You want auth that's quickly set up like Auth0 but open source like Keycloak? Look no further— ZITADEL combines the ease of Auth0 and the versatility of Keycloak.

```

File: README.md [hunk 8]
```diff
@@ -1,112 +1,126 @@
...
+You can also discovery our new pay-as-you-go [pricing](https://zitadel.com/pricing/v2).
 
-## Start using ZITADEL
+### Install ZITADEL
 
-### Quickstarts
+- [We provide installation guides for multiple platforms here](https://docs.zitadel.com/docs/guides/installation)
 
-See our [Documentation](https://docs.zitadel.ch/docs/quickstarts/introduction) to get started with ZITADEL quickly. Let us know, if you are missing a language or framework in the [Q&A](https://github.com/zitadel/zitadel/discussions/1717).
+### Quickstarts - Integrate you app
```
Comment (hifabienne) [inline README.md:54]:
```suggestion
### Quickstarts - Integrate your app
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -12,7 +12,91 @@ Do you miss a feature? Please file an issue [here](https://github.com/zitadel/zi
...
-## Grab an Issues
+## Grab an Issuesa# Contributing to ZITADEL
+
+## Introduction
+
+Thank you for your interest about how to contribute! As you might know there is more than code to contribute. You can find all information needed to start contributing here.
+
+Please give us and our community the chance to get rid of security vularbilities by responsibly disclose this kind of issues by contacting [security@zitadel.com](mailto:security@zitadel.com).
+
+The strongest part of a community is the possibility to share thoughts. That's why we give try to react as soon as possible to your ideas, thoughts and feedback. We love to discuss as much as possible in an open space like in the [issues](https://github.com/zitadel/zitadel/issues) and [discussions](https://github.com/zitadel/zitadel/discussions) section here or in our [chat](https://zitadel.com/chat), but we understand your doubts and provide further contact options [here](https://zitadel.com/contact).
```
[the remark this answers is not in the document]
Comment (fforootd) [reply CONTRIBUTING.md:23]:
```suggestion
The strongest part of a community is the possibility to share thoughts. That's why we try to react as soon as possible to your ideas, thoughts and feedback. We love to discuss as much as possible in an open space like in the [issues](https://github.com/zitadel/zitadel/issues) and [discussions](https://github.com/zitadel/zitadel/discussions) section here or in our [chat](https://zitadel.com/chat), but we understand your doubts and provide further contact options [here](https://zitadel.com/contact).
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -12,7 +12,91 @@ Do you miss a feature? Please file an issue [here](https://github.com/zitadel/zi
 
 Enhancements are discussed and evaluated every Wednesday by the ZITADEL core team.
 
-## Grab an Issues
+## Grab an Issuesa# Contributing to ZITADEL
```
[the remark this answers is not in the document]
Comment (fforootd) [reply CONTRIBUTING.md:15]: something is broken there 👍

**Pull request merged**